webapp/src/Entity/Language.php

```diff
@@ -119,15 +119,19 @@ class Language extends BaseApiEntity
     private Collection $versions;
 
     #[ORM\Column(type: 'blobtext', nullable: true, options: ['comment' => 'Compiler version'])]
+    #[Serializer\Exclude]
     private ?string $compilerVersion = null;
 
     #[ORM\Column(type: 'blobtext', nullable: true, options: ['comment' => 'Runner version'])]
+    #[Serializer\Exclude]
     private ?string $runnerVersion = null;
 
     #[ORM\Column(type: 'string', length: 255, nullable: true, options: ['comment' => 'Runner version command'])]
```

Comment:
```suggestion
    #[OA\Property(nullable: true)]
    #[ORM\Column(type: 'string', length: 255, nullable: true, options: ['comment' => 'Runner version command'])]
```

Regardless of, if we expose this when its null we want to allow that when you set this via the API you don't set this value yet.

Reply:
Isn't `OA\Property` (currently) only used for returning data, not for creating objects? Or I don't get your comment.

Reply:
I might be wrong yes, I meant the one which sets the `nullable` on the model.